src/sources/util/mod.rs

```diff
@@ -6,8 +6,10 @@ mod tcp;
 #[cfg(all(unix, feature = "sources-utils-unix",))]
 mod unix;
 
+#[cfg(any(feature = "sources-http", feature = "sources-logplex"))]
```

Comment:
As mentioned above, we could choose to support this feature with this source. That would alleviate the need to conditionally compile it here.